pandas/tests/tseries/holiday/test_calendar.py

```diff
@@ -81,3 +82,21 @@ def test_calendar_observance_dates():
 def test_rule_from_name():
     us_fed_cal = get_calendar("USFederalHolidayCalendar")
     assert us_fed_cal.rule_from_name("Thanksgiving") == USThanksgivingDay
+
+
+def test_calendar_2031():
+    # See gh-27790
+    #
+    # Labor Day 2031 is on September 1. Saturday before is August 30.
+    # Next working day after August 30 ought to be Tuesday, September 2.
+
+    class testCalendar(AbstractHolidayCalendar):
+        rules = [
+            USLaborDay,
+        ]
+
+    cal = testCalendar()
+    workDay = offsets.CustomBusinessDay(calendar=cal)
+    Sat_before_Labor_Day_2031 = to_datetime('2031-08-30')
+    next_working_day = Sat_before_Labor_Day_2031 + 0 * workDay
+    assert(next_working_day == to_datetime('2031-09-02'))
```

Comment:
And while you're at it: can we move the parentheses here?  We use bare `assert` statements (see the test above).

Reply:
Will do. I'll wait until tomorrow, though. Merging master had no effect just now, so it seems the errors are in the current version.

Reply:
I now get an error message like this:

##[error]Bash exited with code '1'.

But I have no idea what to do about fixing this.

Reply:
https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=17967

So it looks like the `black pandas` lint check is failing (https://pypi.org/project/black/).

Reply:
Reformatted that file using black. Let's see if it works now. I'm hoping it's a bit quicker next time...

Reply:
It passed all tests!